librz/syscall/syscall.c

```diff
@@ -43,6 +74,16 @@ RZ_API void rz_syscall_free(RzSyscall *s) {
 	}
 }
 
+/**
+ * \brief Frees an RzSysregItem type
+ */
+RZ_API void rz_sysreg_item_free(RZ_NONNULL RzSysregItem *s) {
```

Comment:
Usually,  in `_free()` functions we allow an argument to be NULL and just return if it is.